src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -23,16 +23,15 @@ import compose from '../../libs/compose';
 import ONYXKEYS from '../../ONYXKEYS';
 import Picker from '../../components/Picker';
 import AddressForm from './AddressForm';
-import ReimbursementAccountForm from './ReimbursementAccountForm';
-import * as ReimbursementAccount from '../../libs/actions/ReimbursementAccount';
 import * as ReimbursementAccountUtils from '../../libs/ReimbursementAccountUtils';
 import reimbursementAccountPropTypes from './reimbursementAccountPropTypes';
+import Form from '../../components/Form';
 import reimbursementAccountDraftPropTypes from './ReimbursementAccountDraftPropTypes';
 
 const propTypes = {
     /** The bank account currently in setup */
     /* eslint-disable-next-line react/no-unused-prop-types */
-    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
+    reimbursementAccount: reimbursementAccountPropTypes,
```

Comment:
I believe that this one is still required, we should remove `reimbursementAccountDraft` though since that's not always present. Similar to what we did to RequestorStep [here](https://github.com/Expensify/App/pull/12766/files#diff-09392b6dde0a591eb6e22cee1dd6ed5270e6440ae59c7beec01580afb2a5d7ccL32-L35)

Reply:
done